Rewrite the overview section on the docs index page (#121)
IAMconsortium/nomenclature: README.md

> # nomenclature - Working with IAMC-format project templates

Reviewer: What exactly is meant with the term "project template", is that the combination of codelists and model mappings? Do we introduce or explain that somewhere?

Author: Historically speaking, a template is an Excel file with the list of all variables and units that should be reported for a model comparison study.

Reviewer: Ah right, I guess I was missing the historical context on this. So basically what is now called codelists used to be called template?

Reviewer: I'm afraid the mixing of the terms `definitions`, `codelists`, `templates` and `mappings` might still be a little confusing.
The way I currently understand it is as follows: `definitions` = `codelists` and `templates` = `definitions` + `mappings`?
In the first paragraph it says "... manage definitions and 'templates'" while in the second paragraph "A project template consists of "codelists" and mappings". 
As to my understanding the terms codelists and definitions are interchangeable these two sentences are kind of contradictory as we first speak of "definitions and templates" and then explain that "templates are codelists and mappings".
Maybe I'm just mistaken about what is meant my definitions though.

Author: I removed the term "template" entirely, and changed the example of an attribute from "definition" to "description" (we use both terms in some projects).

## Overview

The **nomenclature** package facilitates validation and processing of scenario data

> for model comparison projects and scenario analysis. It allows to manage

Reviewer: Also what are "project templates" (as opposed to codelists)? Do you mean the model mappings?

> for model comparison projects and scenario analysis. It allows to manage

Author: The way that I meant this: the package helps a user to manage the codelists and mappings by e.g., guarding against duplicates, ensuring that units exist for variables, ensuring internal consistency of region mappings

> for model comparison projects and scenario analysis. It allows to manage

Reviewer: Ah right, in that context manage definitely makes sense. Good to keep in all places then.

project templates and "codelists" that follow the format developed by the

> [Integrated Assessment Modeling Consortium (IAMC)](https://www.iamconsortium.org)..

Reviewer: ```suggestion
[Integrated Assessment Modeling Consortium (IAMC)](https://www.iamconsortium.org).
```

A "codelist" is a list allowed values (or "codes") for dimensions of IAMC-format data,
typically *regions* and *variables*. Each code can have additional attributes:

> for example, a "variable" (string) usually has a definition and an expected unit.

Reviewer: This makes it sound like the unit attribute is optional. Maybe something along the lines of:

```suggestion
for example, a "variable" has to have a unit attribute and usually features a definition.
```
would be more clear.

Read the [SDMX Guidelines](https://sdmx.org/?page_id=4345) for more information on
the concept of codelists.

The **nomenclature** package supports three main use cases:

> - Management of codelists, definitions and mappings for model comparison projects

Reviewer: What is the difference between codelists and definitions? Also I'm not sure that nomenclature helps with the management. To me that suggests that nomenclature can somehow create and edit model mappings.

Author: Removed the definition in this list to avoid confusion. For the other question, see the answer above.

Reviewer: Perfect, thank you.
